packages/core/src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -353,5 +358,5 @@ export enum StripeStringConstants {
 export enum StripeElementType {
     PAYMENT = 'payment',
     AUTHENTICATION = 'linkAuthentication',
-    SHIPPING = 'shippingAddress',
+    SHIPPING = 'address',
```

Comment:
I think the key would be ADDRESS instead of SHIPPING
```suggestion
    ADDRESS = 'address',
```

Reply:
Created the key and applied the change, thank you for the comment